src/transformers/modeling_utils.py

```diff
@@ -2217,3 +2217,19 @@ def forward(self, hidden_states):
         return torch.cat(output_chunks, dim=chunk_dim)
 
     return forward_fn(*input_tensors)
+
+
+class IgnoreNoneGetter:
```

Comment:
My 2 cents: rather than the specific (kinda confusing) functionality-driven name, I'd go for just `ModelOutput`.

Reply:
Done